.Rhistory

```diff
@@ -0,0 +1,512 @@
+names <- paste(tab$gene_id,tab$chromosome,tab$genomic_position,sep="_")
```

Comment:
Has this file been committed by accident?

Reply:
Yes, it was added by accident. I have removed it.